crates/rolldown/tests/rolldown/misc/preserve_entry_signature/basic/_config.json

```diff
@@ -6,6 +6,10 @@
     {
       "name": "preserveEntrySignatures/strict",
```

Comment:
You don't really need a name filed if you just want to use `"preserveEntrySignatures": "strict"` as the title. The auto generated-one should already does the trick.

Reply:
that did not work: 

![image](https://github.com/user-attachments/assets/4aa5e453-918b-4307-858e-fc176aee4a7c)

Reply:
Yeah. You need add it manually in https://github.com/rolldown/rolldown/blob/3a38e90233afab38d58c5d0f94fcd30452bcf098/crates/rolldown_testing_config/src/lib.rs#L53-L74

Reply:
Fixed.